sql/core/src/test/scala/org/apache/spark/sql/DynamicPartitionPruningSuite.scala

```diff
@@ -1313,7 +1313,7 @@ abstract class DynamicPartitionPruningSuiteBase
   test("SPARK-32659: Fix the data issue when pruning DPP on non-atomic type") {
     withSQLConf(
       SQLConf.DYNAMIC_PARTITION_PRUNING_FALLBACK_FILTER_RATIO.key -> "2", // Make sure insert DPP
-      SQLConf.DYNAMIC_PARTITION_PRUNING_REUSE_BROADCAST_ONLY.key -> "false") {
+      SQLConf.DYNAMIC_PARTITION_PRUNING_REUSE_BROADCAST_ONLY.key -> "true") {
```

Comment:
why not we just set the broadcast threshold to Long.Max?

Reply:
and then no other configs need to be set.

Reply:
and then no other configs need to be set.

Reply:
fixed